pkgs/top-level/all-packages.nix

```diff
@@ -31049,10 +31049,10 @@ with pkgs;
 
   jwm-settings-manager = callPackage ../applications/window-managers/jwm/jwm-settings-manager.nix { };
 
-  k3s_1_23 = callPackage ../applications/networking/cluster/k3s/1_23 { };
-  k3s_1_24 = callPackage ../applications/networking/cluster/k3s/1_24 { };
-  k3s_1_25 = callPackage ../applications/networking/cluster/k3s/1_25 { };
-  k3s_1_26 = callPackage ../applications/networking/cluster/k3s/1_26 { };
+  # Kubernetes version support matrix: https://endoflife.date/kubernetes
+  k3s_1_24 = callPackage ../applications/networking/cluster/k3s/1_24 { }; # Remove after 2023-07-28
+  k3s_1_25 = callPackage ../applications/networking/cluster/k3s/1_25 { }; # Remove after 2023-10-27
```

Comment:
Looks like these two versions would go EOL before 23.11 would be released (end of November).

Reply:
NixOS's users need to migrate to the latest version provided during a release to ensure the security of their deployment anyway, so what's the big deal? 

Are you saying that everyone is running out of date releases of k3s?

Reply:
> so what's the big deal?

Communication.

> Are you saying that everyone is running out of date releases of k3s?

I don't know.

As requested at Matrix. Will remove versions k3s_1_24 and k3s_1_25 too.